packages/mobile_app/lib/components/small_card.dart

```diff
@@ -4,9 +4,15 @@ import 'package:my_prophet/services/static_assets.dart';
 const _height = 130.0;
 const _width = 64.0;
 
+enum SmallCardMode {
+  CHOSEN,
+  WASCHOSEN,
+  INTACT,
+}
+
 class SmallCard extends StatefulWidget {
```

Comment:
What is *small card* and *big card*? The naming is confusing IMHO. If we are naming *cards* after elements on the main screen, then we might name Taro cards as *Taro*